### ISSUE TITLE
Some links and tabs are missing the necessary labels and roles for accessibility

### ISSUE THREAD

**Issue by Piciok** [BeWelcome/rox#116]
On the main site after logging in, certain links are not labelled and in one place a switch does not indicate its role.

The links on the main site leading to the inbox, logging out and the one expanding a menu with profile editing, groups, volunteering etc. are unlabelled which for screenreader users means that, although the information of a link being there is given, it is not announced what it does.

Additionally, the switch allowing to change between showing all messages and unread messages does not carry any control role such as tab or radio button so it is impossible to tell using a screenreader which setting is currently selected.

Fixes of those issues would be highly appreciated.

**Comment by amnesiac84** [comment]
I hope I fixed it with this commit: https://github.com/BeWelcome/rox/commit/254b797a90e2baf41721aca186c3ef778457979e
I'm still learning about accessibility, so please do comment when there are still/more issues

**Comment by amnesiac84** [comment]
Available on https://beta.bewelcome.org/

**Comment by TillWenke** [comment]
@Piciok should be fixed :)

**Comment by buyolitsez** [comment]
@thisismeonmounteverest seems fixed
